Platforms/LangCSharp/PlatformImpl.cs

```diff
@@ -147,7 +147,9 @@ public override string GenerateCodeForFunction(AbstractTranslator translator, Fu
             output.Append(this.NL);
             output.Append("{");
             output.Append(this.NL);
-            translator.TabDepth = 1;
+			translator.TabDepth = 1;
+			output.Append("#pragma warning disable 0219"); // Added by TimJSwan89 to avoid warnings for unused generated variables.

```

Comment:
I want to go through and fix each of these since an unused variable in translated code means there's an unused variable in the original Pastel code that needs to be addressed. Mostly these are artifacts from when the Pastel code was generated itself that need to be removed. But I can see how these can be annoying, so it'll be good for others that this is here until I have time to go through those. 

Also, you don't need to add the attribution part of the comment, since that's what git blame/history is for.